lib/mail-archive-helper.ts

```diff
@@ -128,14 +128,26 @@ export class MailArchiveGitHelper {
                     const pre = info?.text
                         .replace(/&/g, "&amp;")
                         .replace(/</g, "&lt;").replace(/>/g, "&gt;");
-                    const comment = `There was a [status update](${
-                        whatsCookingBaseURL}${
-                        sousChef.messageID}) in the "${
-                        info?.sectionName}" section about the branch [\`${
-                        branchName}\`](${
-                        branchBaseURL}${
-                        branchName}) on the Git mailing list:\n\n<pre>\n${
-                        pre}\n</pre>`;
+                    let comment;
+                    if (!pre || pre.trim() === "") {
```

Comment:
Did you want to add a test for this comment being generated?

Reply:
Not really... :-)